docs/sources/whatsnew/whats-new-in-v10-0.md

```diff
@@ -0,0 +1,277 @@
+
+Correlations enable you to seamlessly jump from one data source to another. You define relationships between your different data sources, and when Exploring simply click a button next to a related field in one data source and Grafana will run the corresponding query in the other datasource.
+
+It's worth noting that Correlations is currently in preview. As such, it's recommended to enable it only on test or development instances, rather than in production environments.
+
+To try out Correlations, you'll need to enable the `correlations` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
+
+In subsequent releases, we’ll be refining and enhancing the user interface for Correlations, to provide a more streamlined user experience.
+
+## Enhanced Data Source Selection Experience: Simplifying Querying for Users
+
+_Available in preview in all editions of Grafana._
+Concepts like data sources and dashboards panels are hard to grasp and it’s a struggle to go from Grafana’s “empty state” to a working dashboard that displays data. Our latest advancements streamline the process of selecting the ideal data source in Grafana, prioritizing recent usage and providing labels and supplementary descriptions.
+
+With this flow, selecting a data source has been greatly simplified, providing a clear overview of available data sources and allowing users to quickly connect to a new one when needed. Additionally, the flow now enables quick uploading of CSV files and access to built-in data sources.
+
+<!--
+- TODO:Add screenshots
+-->
+
+To try it out, you'll need to enable the `advancedDataSourcePicker` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
+
+## Dashboards and visualizations
+
+### Public dashboards
+
+_Available in preview in all editions of Grafana._
+Public dashboards allow you to share your Grafana dashboard with anyone without them having to log into Grafana. This is useful when you want to expose your dashboard to the world.
+
+These are some of the improvements you will find in this version:
+
+- The time picker and annotations can be toggled on or off from the configuration modal
+- You can see all the public dashboards from the Dashboards menu
+- Improved UI: new modal design, paused, and not found pages
+- Bug fixes! E.g collapsed rows, hidden queries, and zoom into panels
+
+To try it out, you'll need to enable the `publicDashboards` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
+
+To learn more, check out our public dashboards [documentation](https://grafana.com/docs/grafana/latest/dashboards/dashboard-public/?pg=blog&plcmt=body-txt).
```

Comment:
We should be using relrefs rather than absolute links so that they always resolve no matter what version we point to
```suggestion
To learn more, check out our public dashboards [documentation]({{< relref "../../dashboards/dashboard-public" >}}).
```